### PR TITLE
Map Render Service

### PR DESCRIPTION
**Pull request by purdyk**
for creating really big mapz broz.
yeah

### CODE REVIEW

File: renderer/MapRenderer.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import mapnik
+
+
+class MapRenderer:
+    def __init__(self):
+        # style = '/var/map-styles/ridewithgps-tilemill/ridewithgps-tilemill.xml'
+        self.style = '/vagrant/renderer/rwgps.xml'
```
Comment (kingcu) [inline renderer/MapRenderer.py]: it doesn't parse out the projection from the style as a default?

Comment (kingcu) [inline renderer/MapRenderer.py]: needs a config file where we can specify diff style paths for production vs dev

File: renderer/static/root.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+var load_image = function() {
```
Comment (kingcu) [inline renderer/static/root.js]: var?!?!?! where's babel + webpack + babzoozal?

Comment (purdyk) [inline renderer/static/root.js]: This is running in its own micro service.
I don't have time for webpack.

File: renderer/MapRenderer.py (hunk 1 above)
Comment (purdyk) [inline renderer/MapRenderer.py]: This is resolved.
Should have just shipped with prod. Setting up dev environment is a pain in the ass.

**Pull request merged**